odoo/models.py

```diff
@@ -3109,7 +3116,18 @@ def _read_format(self, fnames, load='_classic_read'):
         data = [(record, {'id': record._ids[0]}) for record in self]
         use_name_get = (load == '_classic_read')
         for name in fnames:
-            convert = self._fields[name].convert_to_read
+            field = self._fields[name]
+            if field.type == 'properties':
+                values_list = [record[name] for record, _ in data]
+                records = self.browse([record.id for record, _ in data])
+
+                results = field.convert_to_read_multi(values_list, records, use_name_get)
```

Comment:
What about the `try: except MissingError:` ? If you read a non-existing record, it will crash instead of returning nothing for this record (current behavior). Or I miss something ?

Reply:
indeed, I updated the code and added a small test to be sure it doesn't crash in that case